lib/l10n/app_pt.arb

```
@@ -337,5 +338,7 @@
   },
   "appLogsFilteredCount": "{shown} de {total}",
   "invalidRelayUrlScheme": "O URL deve começar com wss:// ou ws://",
-  "invalidRelayUrl": "URL de relay inválida"
+  "invalidRelayUrl": "URL de relay inválida",
+  "helpAndFeedback": "Help and feedback",
```

RuboCop (1.85.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)